webapi/_test_data/ontologies/beol-onto.ttl

```diff
@@ -425,6 +442,42 @@
                    rdfs:subPropertyOf knora-base:hasValue .
 
     ### ###########################################
+    ### beol:hasTitle
+
+    :hasTitle rdf:type owl:ObjectProperty ;
```

Comment:
well, your beol:title is for title of a letter/manuscript, beol:hasTitle is for title of a person. 

I can change the beol:hasTite to beol:personHasTitle and change beol:title for letter and manuscript to beol:hasTitle. How does that sound?